src/videojs.ads.js

```diff
@@ -246,6 +268,37 @@ var
       // just resume playback at the current time.
       player.play();
     }
+
+    i = suppressedTracks.list.length;
+    if (tracks.addTrack_) {
+      while (i--) {
+        track = suppressedTracks.list[i];
+        tracks.addTrack_(track);
```

Comment:
I debated using `.addTrack_` here a lot. One of the issues with not using it is that currently we don't store the associated src on the TextTrack objects themselves but if we did and we recreated the tracks, we would need to make another request (though, it should be cached) and parse the vtt file again, so, I thought just stuffing it back in was reasonable, if not great.
The main reason that `addTrack_` is pseudo-private is that we need a way to add items to the list internally but it isn't part of the official API spec. Maybe, we should just make them official APIs?